### PR TITLE
New package: libtommath-1.0

### LINKED ISSUE #3537: Package request: libtommath
Vendored by MoarVM and and libtorrent-rasterbar, we should package this ourself.

### PR DESCRIPTION
**Pull request by ebfe**
Closes #3537

### CODE REVIEW

File: srcpkgs/libtommath/template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+hostmakedepends="libtool"
+short_desc="A portable number theoretic multiple-precision integer library"
+maintainer="Michael Gehring <mg@ebfe.org>"
+license="Public domain"
+homepage="http://www.libtom.org/LibTomMath/"
+distfiles="https://github.com/libtom/libtommath/releases/download/v${version}/ltm-${version}.tar.xz"
+checksum=993a7df9ee091fca430cdde3263df57d88ef62af8103903214da49fc51bbb56c
+
+do_build() {
+	if [ "$CROSS_BUILD" ]; then
```
Comment (Vaelatern) [inline srcpkgs/libtommath/template]: Why just for cross building and not for all builds?

Comment (ebfe) [inline srcpkgs/libtommath/template]: Good question. Mostly because this is c/p from libtomcrypt/template and wasn't necessary for non-cross builds.

**Comment by xtraeme** [comment]
:+1:

**Comment by ebfe** [comment]
Merged in 45fa33fb4598587faba7051b59731d20934266d1

**Pull request closed without merging**